pipeline/src/helpers/prismic.ts

```diff
@@ -17,8 +17,9 @@ const PRISMIC_MAX_PAGE_SIZE = 100;
 
 type GetPrismicDocumentsParams = {
   publicationWindow: TimeWindow;
-  graphQuery?: string;
+  graphQuery: string;
   after?: string;
+  documentType?: string;
```

Comment:
Can `documentType` ever be undefined?

Reply:
It could, when this prismic helper was used by the 2 different ETL pipelines. I belatedly came to the realisation that the existing pipeline could do the job so that's gone now 🙃